Assets/Scripts/Game/MagicAndEffects/Effects/Illusion/InvisibilityNormal.cs

```diff
@@ -19,25 +19,41 @@ namespace DaggerfallWorkshop.Game.MagicAndEffects.MagicEffects
     /// </summary>
     public class InvisibilityNormal : ConcealmentEffect
     {
+        public static readonly string PublicKey = "Invisibility-Normal";
+
         public override void SetProperties()
         {
-            properties.Key = "Invisibility-Normal";
+            properties.Key = PublicKey;
```

Comment:
Here's an example of making the EntityEffect key publicly available to support multi-effect potions while still keeping all potion recipes in EntityEffect classes as you wanted. We should discuss if there's a better way of doing this.

Reply:
I'm happy with this for now. Thank you for overcoming this problem. Let's see how it goes before we worry about refining. 👍

Reply:
Okay should I change all of the entity effect classes to follow the same pattern then? Think I will change the name from PublicKey to EffectKey though.

Reply:
I'm happy for that, if you'd like to work through it. I'll make sure I follow the same pattern for any future effects that I write.

BTW, now you're across the effect system would you be interested in helping to build any of the remaining spell effects, item effects, advantages/disadvantages, or artifact effects? I plan to start knocking these over again soon once I'm done having fun with vampirism.